.github/workflows/pr_created.yaml

```diff
@@ -0,0 +1,36 @@
+name: PR Created
+
+on:
+  pull_request:
+    types: [opened, reopened]
```

Comment:
Will this trigger for sdraft PRs? Ideally draft PRs would be ignored and this would only trigger when the PR is made non-draft.

Reply:
this will not include drafts as it stands. 

`[opened, reopened, ready_for_review]` would include them